Css/style.css

```diff
@@ -285,12 +343,15 @@ button {
 
 .btn {
   padding: 12px;
-  background: #36b37f;
+  background: var(--primary);
   border-radius: 4px;
   font-weight: 500;
   font-size: 17px;
   line-height: 24px;
   color: #fff;
+  transition: all 0.5s ease-in-out;
+  position: relative;
+  z-index: -1;
```

Comment:
- [ ] That we specified `z-index: -1;` makes our button behind `li` so that we can not any longer click the button, I assume that was not intended, let's make sure we fix that 😄